common/src/main/java/com/wynntils/core/config/ConfigManager.java

```diff
@@ -129,10 +129,12 @@ public static void saveConfig() {
 
     private static List<ConfigHolder> collectConfigOptions(Feature feature) {
         FeatureInfo featureInfo = feature.getClass().getAnnotation(FeatureInfo.class);
-        // feature has no category defined, can't create options
-        if (featureInfo == null || featureInfo.category().isBlank()) return null;
 
-        String category = featureInfo.category();
+        String category = "";
+        // feature has no category defined, can't create options
```

Comment:
i think you forgot to change this comment? since it's no longer true that we ignore options